RecoTauTag/Configuration/python/tools/DisplayManager.py

```diff
@@ -0,0 +1,141 @@
+import ROOT
+import copy
+import math
+
+def applyLegendSettings(leg):
+    leg.SetBorderSize(0)
+    leg.SetFillColor(10)
+    leg.SetLineColor(0)
+    leg.SetFillStyle(0)
+#   leg.SetTextSize(0.035)
+#   leg.SetTextFont(42)
+
+def createRatioCanvas(name, errorBandFillColor=14, errorBandStyle=3354):
+    cv = ROOT.TCanvas(name.replace('.pdf', ''), name.replace('.pdf', ''), 10, 10, 700, 600)
+    ## this is the tricky part...
+    ## Divide with correct margins
+    cv.Divide(1, 2, 0.0, 0.0)
+    ## Set Pad sizes
+    cv.GetPad(1).SetPad(0.0, 0.32, 1., 1.0)
+    cv.GetPad(2).SetPad(0.0, 0.00, 1., 0.34)
+    cv.GetPad(1).SetFillStyle(4000)
+    cv.GetPad(2).SetFillStyle(4000)
+    ## Set pad margins 1
+    cv.cd(1)
+    ROOT.gPad.SetTopMargin(0.08)
+    ROOT.gPad.SetLeftMargin(0.12)
+    ROOT.gPad.SetBottomMargin(0.03)
+    ROOT.gPad.SetRightMargin(0.1)
+    ## Set pad margins 2
+    cv.cd(2)
+    ROOT.gPad.SetBottomMargin(0.35)
+    ROOT.gPad.SetLeftMargin(0.12)
+    ROOT.gPad.SetRightMargin(0.1)
+
+    bogyHist = ROOT.TH1F("legendPseudoHist", "", 1, 1., 2.)
+    bogyHist.SetFillColor(errorBandFillColor)
+    bogyHist.SetFillStyle(errorBandStyle)
+    bogyHist.SetLineColor(0)
+    
+    cv.cd(1)
+    return cv
+
+def checkDifferences(histos):
+    rel_diff = 0
+    if len(histos)>0:
+        for ib in range(0, histos[0].GetNbinsX()):
+            for ih, h in enumerate(histos):
+                if not ih == 0:
+                    if not histos[0].GetBinContent(ib) == 0:
+                        rel_diff+=((h.GetBinContent(ib)-histos[0].GetBinContent(ib))/histos[0].GetBinContent(ib))*((h.GetBinContent(ib)-histos[0].GetBinContent(ib))/histos[0].GetBinContent(ib))
```

Comment:
you may want to apply the bugfix from https://github.com/cms-sw/cmssw/commit/8ebac5517863086c98e55e3092f13ef0cb6be289

```diff
             for ih, h in enumerate(histos):
                 if not ih == 0:
-                    if not histos[0].GetBinContent(ib) == 0:
-                        rel_diff+=((h.GetBinContent(ib)-histos[0].GetBinContent(ib))/histos[0].GetBinContent(ib))*((h.GetBinContent(ib)-histos[0].GetBinContent(ib))/histos[0].GetBinContent(ib))
+                    if not histos[0].GetBinContent(ib+1) == 0:
+                        rel_diff+=((h.GetBinContent(ib+1)-histos[0].GetBinContent(ib+1))/histos[0].GetBinContent(ib+1))*((h.GetBinContent(ib+1)-histos[0].GetBinContent(ib+1))/histos[0].GetBinContent(ib+1))
     return math.sqrt(rel_diff)
```